<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Config dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>201</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="envTab">
    <attribute name="title">
     <string>Environment</string>
    </attribute>
    <widget class="QLabel" name="backgrounSetupLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>96</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Setup background</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bgPathLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="browseBgButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse ...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resetBgButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QLabel" name="setupBgMusicLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>132</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Setup background music</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="bgMusicPathLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="browseBgMusicButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse ...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resetBgMusicButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>140</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="gameTab">
    <attribute name="title">
     <string>Game</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>236</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="nullificationLabel">
        <property name="text">
         <string>Nullification count down</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="nullificationSpinBox">
        <property name="suffix">
         <string> seconds</string>
        </property>
        <property name="minimum">
         <number>5</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="value">
         <number>8</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
